.github/workflows/maven.yml

```diff
@@ -24,5 +24,5 @@ jobs:
           java-version: ${{ matrix.java }}
       - name: Test with Maven
         env:
-          MAVEN_OPTS: "-Xms3g -Xmx3g"
+          MAVEN_OPTS: "-Xms3g -Xmx9g"
```

Comment:
9GB??? woah

Reply:
Yes it takes a lot of memory for compiler and for javadoc. It's a beast